bin/build_extra_package.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+require 'manageiq/rpm_build/extra_package'
+
+Dir.chdir(ManageIQ::RPMBuild::ExtraPackage::PACKAGES_DIR) do
```

Comment:
This chdir feels unnecessary if it's only used for the Dir glob.

Reply:
If I don't chdir, the directories will be fully qualified paths which will require further processing.